### PR TITLE
docs: collect token API from tsdoc

### LINKED ISSUE #38691: 定制主题 属性不全
### Reproduction link

[https://ant.design/docs/react/customize-theme-v5-cn#seedtoken](https://ant.design/docs/react/customize-theme-v5-cn#seedtoken)

### Steps to reproduce

使用theme修改属性  个别属性提示不存在 例如 radiusBase

### What is expected?

改变样式（举例：圆角改变）

### What is actually happening?

提示没有 该属性  
SeedToken文档 api中存在该属性（）

| Environment | Info |
| --- | --- |
| antd | 5.0.0 |
| React | react 18.2.0 |
| System | windows |
| Browser | chrom |

---
不只圆角这一个 应该是文档和实际api未同步

### PR DESCRIPTION
**Pull request by MadCcc**
- chore: token doc
- docs: token meta

[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [x] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 🔗 Related issue link
close https://github.com/ant-design/ant-design/issues/38691

### 💡 Background and solution
Collect token API from ts doc automaticly. Result will be in `version/token-meta`.

Run `npm run token-meta` to update token meta manually.

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |      -     |
| 🇨🇳 Chinese |    -       |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

### CODE REVIEW

File: scripts/generate-token-meta.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
+const TypeDoc = require('typedoc');
```
Comment (zombieJ) [inline scripts/generate-token-meta.js:1]: 牛了

**Review by zombieJ — APPROVED** [review]

**Review by afc163 — APPROVED** [review]

File: .dumi/theme/builtins/TokenTable/index.tsx [hunk 2]
```diff
@@ -0,0 +1,114 @@
+import React, { FC, useMemo } from 'react';
+import tokenMeta from 'antd/es/version/token-meta.json';
```
Comment (afc163) [inline .dumi/theme/builtins/TokenTable/index.tsx:2]: 这个文件有多大，会不会发布到 npm，会不会增加 install size。

Comment (MadCcc) [reply .dumi/theme/builtins/TokenTable/index.tsx:2]: 会发布到 npm，给编辑器用，应该还挺大的，所有的 token 信息都会统计进去

Comment (afc163) [reply .dumi/theme/builtins/TokenTable/index.tsx:2]: 会有多大预估一下。普通用户用不到的话，感觉不应该放进去？

antd 目前的 installsize 在主流组件库里已经算比较大的了 https://packagephobia.com/result?p=antd

Comment (MadCcc) [reply .dumi/theme/builtins/TokenTable/index.tsx:2]: 现在是 19 KB，添加描述后会更大。
不过 dts 会保留 tsdoc，这个可以先不添加在 npm 包里。

**Comment by afc163** [comment]
19KB还好

**Pull request merged**